deps/wazuh_testing/wazuh_testing/tools/agent_simulator.py

```diff
@@ -703,6 +909,66 @@ def get_message(self, event_type=None):
         return self.generate_message()
 
 
+class GeneratorHostinfo:
+    def __init__(self):
+        self.HOSTINFO_MQ = 3
+        self.hostinfo_basic_template = 'Host: <random_ip> (), open ports: '
+        self.protocols_list = ['udp', 'tcp']
+        self.localfile = '/var/log/nmap.log'
+
+    def generate_event(self):
+        number_open_ports = randint(1, 10)
+        host_ip = random_ip()
+        message_open_port_list = ''
+        for i in range(number_open_ports):
+            message_open_port_list += fr"{randint(1,65535)} ({choice(self.protocols_list)}) "
+
+        message = self.hostinfo_basic_template.replace('<random_ip>', host_ip)
+        message += message_open_port_list
+        message = fr"{self.HOSTINFO_MQ}:{self.localfile}:{message}"
+        return message
```

Comment:
```suggestion

        return message
```

Reply:
This is unresolved yet

Reply:
This was done by @Rebits here https://github.com/wazuh/wazuh-qa/commit/6976fb26ae1e4fcd0d905a309eb1820db1111a5c